src/main/kotlin/calculator/domain/Operand.kt

```diff
@@ -0,0 +1,19 @@
+package calculator.domain
+
+private const val ZERO = 0
+
+data class Operand(val value: Int) {
+    init {
+        if (value < ZERO) {
+            throw RuntimeException("음수는 입력할 수 없습니다.")
+        }
+    }
+
+    constructor(value: String) : this(
+        if (!value.matches(Regex("[0-9]+"))) {
```

Comment:
1. `Regex("[0-9]+")` 를 상수로 정의해서 사용하는 것이 매번 생성하는 것보다 비용 측면에서 좋겠네요~

2. 다음과 같이 간략하게 표현 가능할 거 같습니다!
```suggestion
        if (!value.matches(Regex("\\d+"))) {
```

Reply:
네, 조언해 주신 대로 정규식 패턴을 상수로 변경하였습니다! 상수를 탑레벨에 선언할지 `companion object`에 둘지 고민하다가 일단 지금까지 하던대로 탑레벨에 선언하였는데요! 혹시 잘못된 부분이 있다면 확인 부탁드리겠습니다! 😃 

- https://github.com/next-step/kotlin-lotto/commit/3f6fab4b3c8183e25839ba583759d43d723cf715